### PR TITLE
Add README for Track and Trace

### PR DESCRIPTION
**Pull request by RyanLassigBanks**
Adds README to track and trace project, detailing how to run the project locally.

Signed-off-by: Ryan Banks <rbanks@bitwise.io>

### CODE REVIEW

File: track_and_trace/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Track and Trace
+
+The Grid Track and Trace transaction family allows users to track goods as they move through a supply chain. Records for goods include a history of ownership and custodianship, as well as histories for a variety of properties such as temperature and location. These properties are managed through a user-specifiable system of record types. This readme  outlines how to start an instance of Track and Trace locally.
```
Comment (agunde406) [inline track_and_trace/README.md]: Please wrap the lines at 80 characters.

Comment (chenette) [inline track_and_trace/README.md]: Delete the extra space after "This readme"

Also, README is usually written in all caps.

File: track_and_trace/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Getting Started
+
+1) Clone the [grid](https://github.com/hyperledger/grid) and
+[grid-contrib](https://github.com/hyperledger/grid-contrib) repositories from github.
+    ```
+    $ git clone https://github.com/hyperledger/grid.git
+    $ git clone https://github.com/hyperledger/grid-contrib.git
+    ```
+2) Navigate to the Track and Trace directory in the grid-contrib repository and run docker-compose.
```
Comment (agunde406) [inline track_and_trace/README.md]: This line is no longer accurate with the change to move to the grid repo instead of grid-contrib

**Comment by vaporos** [comment]
We should differentiate between the platform track and trace capability and the example app in contrib.

**Comment by RyanLassigBanks** [comment]
> We should differentiate between the platform track and trace capability and the example app in contrib.

Added the following to the start of the README

>This README outlines how to start the
example Track and Trace applications AssetTrack and FishNet locally using
Docker.

@vaporos Is there any specific wording you'd like to see or is the above acceptable?

File: track_and_trace/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+1) Clone the [grid](https://github.com/hyperledger/grid) and
+[grid-contrib](https://github.com/hyperledger/grid-contrib) repositories from github.
+    ```
+    $ git clone https://github.com/hyperledger/grid.git
+    $ git clone https://github.com/hyperledger/grid-contrib.git
+    ```
+2) Navigate to the grid directory and run docker-compose.
+    ```
+    $ cd grid
+    $ docker-compose -f docker-compose.yaml -f ../grid-contrib/track-and-trace/docker-compose.yaml up --build
```
Comment (agunde406) [inline track_and_trace/README.md]: `../grid-contrib/track-and-trace/docker-compose.yaml` should be`../grid-contrib/track_and_trace/docker-compose.yaml`

**Pull request merged**